soroban-env-common/src/arbitrary.rs

```diff
@@ -0,0 +1,48 @@
+//! Implementations of [`Arbitrary`] for contract types.
+
+#![cfg(feature = "testutils")]
+
+extern crate alloc;
+
+use crate::xdr::ScStatus;
+use crate::Status;
+use crate::SymbolSmall;
+use alloc::string::String;
+use alloc::vec::Vec as RustVec;
```

Comment:
There is no name collision here with some other `Vec`, is there? Up to you though.

Reply:
I've removed it.